pilot/pkg/xds/endpoint_builder.go

```diff
@@ -140,6 +150,12 @@ func (b EndpointBuilder) DependentConfigs() []model.ConfigKey {
 	return configs
 }
 
+var edsDependentTypes = []config.GroupVersionKind{gvk.PeerAuthentication}
```

Comment:
Can we avoid global variable?

Reply:
I don't think having a global constant is generally an issue? we have this all over the place. Its mostly mutable global vars that are a concern afaik?